src/AppWindow.cpp

```diff
@@ -28,55 +29,75 @@ AppWindow::AppWindow(BRect frame)
 	MainView = new AppView(Bounds());
 	AddChild(MainView);
 	
-	PassOut = new RandoTextView(BRect(5,25,200,40), "PassOut",
-		BRect(0,0,200,30), be_bold_font, NULL, B_FOLLOW_LEFT|B_FOLLOW_TOP,
+	// Result Label:
+	BStringView *result = new BStringView(BRect(100,30,11,11),"resultlabel","RESULT");
+	result->ResizeToPreferred();
+	MainView->AddChild(result);
+	
+	// Result:
+	PassOut = new RandoTextView(BRect(5,50,245,80), "PassOut",
+		BRect(5,0,245,60), be_bold_font, NULL, B_FOLLOW_LEFT|B_FOLLOW_TOP,
```

Comment:
It's better to change `RandoTextView` to a `BTextControl` if we want to add a label to the `PassOut` box, but then the window would be too wide. I suggest that we hold off this change until after the layout of the window is redesigned.